.github/workflows/moac.yaml

```diff
@@ -0,0 +1,30 @@
+name: "moac"
+on:
+  pull_request:
+    paths:
+      - 'csi/moac/**'
+      - 'rpc/proto/**'
+jobs:
+  Build:
+    name: Build and run moac tests
+    runs-on: ubuntu-latest
+    container:
+      image: docker.io/mayadata/ms-buildenv:nix
+    steps:
+      - uses: actions/checkout@v2
+      # npm prepare is normally done by npm install but not if run as a root
+      - run: cd csi/moac && npm install && npm run-script prepare
```

Comment:
we don't currently run `npm run-script prepare` in the build yaml. Why is it needed here? Should it be also in build.yaml?

Reply:
I assume you mean gitlab.yaml. In build.yaml we didn't build moac at all (only the docker image  using nix). "run-script prepare" is needed because when npm install is run as root (which is the case here), it does not run installation scripts and we have an installation script that makes a copy of grpc proto files in moac's dir. In gitlab.yaml we use a different trick and that is "npm install --unsafe-perm" that runs the scripts even if run as root. So two different tricks to overcome the npm quirk. One cannot say that one is better than the other.